src/Store/ThemeStoreConnector.re

```diff
@@ -101,20 +100,8 @@ let start = (themeInfo: ThemeInfo.t) => {
       )
     );
 
-  let loadThemeByPathEffect = (uiTheme, themePath) =>
-    Isolinear.Effect.createWithDispatch(
-      ~name="theme.loadThemeByPath", dispatch =>
-      loadThemeByPath(uiTheme, themePath, dispatch)
-    );
-
-  let loadThemeByNameEffect = name =>
-    Isolinear.Effect.createWithDispatch(
-      ~name="theme.loadThemeByName", dispatch =>
-      loadThemeByName(name, dispatch)
-    );
-
   let onChanged = (newTheme, dispatch) =>
-    loadThemeByName(newTheme, dispatch);
+    dispatch(Actions.ThemeChanged(newTheme));
```

Comment:
Instead of callbacks invoking side-effects directly, dispatch a message and encapsulate the side-effect in an `Effect`.